tests/create_bucket_v2/main.tf

```diff
@@ -0,0 +1,27 @@
+provider "aws" {
+  alias   = "v2"
```

Comment:
remove this line, no need to set an alias.

Reply:
Removed